shared/util.go

```diff
@@ -30,6 +30,24 @@ const StableLabel = "stable"
 // i.e. run from the master branch.
 const MasterLabel = "master"
 
+// ProductChannelToLabel maps known product-specific channel names
+// to the wpt.fyi model's equivalent.
+func ProductChannelToLabel(channel string) string {
```

Comment:
BTW, I just sent out #844 , which will make this mapping no longer necessary.

Reply:
More accurately, not necessary after https://github.com/web-platform-tests/wpt.fyi/issues/611 is closed (because the data is backfilled).